boards/arm/mps2_an521/Kconfig.defconfig

```diff
@@ -15,6 +15,13 @@ config BOARD
 	default "mps2_an521_nonsecure" if TRUSTED_EXECUTION_NONSECURE
 	default "mps2_an521"
 
+# By default, if we build for a Non-Secure version of the board,
+# force building with TF-M as the Secure Execution Environment.
+
+config BUILD_WITH_TFM
+	default y if TRUSTED_EXECUTION_NONSECURE
+
```

Comment:
Should we be do this for all "nonsecure" board configs?

Reply:
I do not have a strong opinion.
I did this for the qemu target, as TFM is the "official" SPE solution for Zephyr. But, I would not "force" the other vendors (STM, NXP) to use TFM by default when building their non-secure applications.